lib/improver/tests/spotdata/spotdata/test_ancillaries.py

```diff
@@ -0,0 +1,175 @@
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Unit tests for the spotdata.ancillaries."""
+
+
+import unittest
+import numpy as np
+from iris.tests import IrisTest
+from subprocess import call as Call
+from tempfile import mkdtemp
+import iris
+from iris.coords import DimCoord
+from iris.cube import Cube
+
+from improver.spotdata.ancillaries import get_ancillary_data as Plugin
+
+
+class Test_get_ancillary_data(IrisTest):
+
+    """
+    Test the reading of ancillary data files amd creation of an ancillaries
+    dictionary.
+
+    """
+
+    def setUp(self):
+        """
+        Create a cube containing a regular lat-lon grid and other necessary
+        ingredients for unit tests.
+
+        """
+        data = np.zeros((20, 20))
+        latitudes = np.linspace(-90, 90, 20)
+        longitudes = np.linspace(-180, 180, 20)
+        latitude = DimCoord(latitudes, standard_name='latitude',
+                            units='degrees')
+        longitude = DimCoord(longitudes, standard_name='longitude',
+                             units='degrees')
+
+        cube = Cube(data,
+                    long_name="template",
+                    dim_coords_and_dims=[(latitude, 0), (longitude, 1)],
+                    units="1")
+
+        orography = cube.copy()
+        orography.data[0:10, 0:10] = 2.
+        orography.rename('surface_altitude')
+        land = cube.copy()
+        land.rename('land_binary_mask')
+        land.data = land.data + 1
+
+        self.orography = orography
+        self.land = land
+        self.directory = mkdtemp()
+        self.orography_path = self.directory + '/orography.nc'
+        self.land_path = self.directory + '/land_mask.nc'
```

Comment:
They could, but I prefer unit tests to be completely standalone. That way they can move with their related routine and nothing else needs to be copied with them.